### DISCUSSION TITLE
How to find the file 'en-us.json' for deploying with docker?

### DISCUSSION THREAD

**Discussion by Cheyenne-Zhang** [open-metadata/OpenMetadata#13011] [Q&A]
Hello, where is the language configuration file 'en-us.json' for deploying with docker?

**Comment by harshach** [comment]
en-us translation is here https://github.com/open-metadata/OpenMetadata/tree/main/openmetadata-ui/src/main/resources/ui/public/locales/en-US
what specifically are you looking for?
